src/components/accessories/patientOPD/patientOPDForm/PatientOPDForm.tsx

```diff
@@ -86,7 +109,10 @@ const PatientOPDForm: FunctionComponent<TProps> = ({
   const diseases = useSelector<IState, DiseaseDTO[]>(
     (state: IState) => state.diseases.diseasesOpd.data ?? []
   );
-
+  const username = useSelector(
+    (state: IState) => state.main.authentication.data?.username
+  );
+  const [operationRows, setOperationRows] = useState([] as OperationRowDTO[]);
```

Comment:
I think that we should implement some kind of initialization of those rows when we are in edit mode.
If I try to edit a previous opd that has a relationship with some operations we should display in this form.

Reply:
Done